recipes/skll/meta.yaml

```diff
@@ -0,0 +1,88 @@
+package:
+  name: skll
+  version: 2.5
```

Comment:
Please check our example recipe on how to format the version and used it as a variable in the url.

Reply:
Thanks @ocefpaf. Done!